code/game/gamemodes/cult/talisman.dm

```diff
@@ -33,9 +33,9 @@
 				return
 			if("supply")
 				supply()
-		user.take_organ_damage(5, 0)
-		if(src && src.imbue!="supply" && src.imbue!="runestun")
-			if(delete)
+		if(ishuman(user))

```

Comment:
Change this block up until the return to look like this: (MAKE SURE TO CHANGE SPACE BASED INDENTATION TO TAB BASED)
```dm
		user.apply_damage(10, 15)
		if (src && src.imbue != "supply" && src.imbue != "runestun" && delete)
			qdel(src)
```